packages/devextreme/testing/tests/DevExpress.utils/utils.date.tests.js

```diff
@@ -0,0 +1,76 @@
+import dateUtils from 'core/utils/date';
+const { test, module } = QUnit;
+
+module('getRangesByDates', {}, () => {
+    test('getRangesByDates should generate correctly ranges when dates is empty array', function(assert) {
```

Comment:
[minor]
In each test
```suggestion
    test('getRangesByDates should generate correct ranges when dates is empty array', function(assert) {
```

Reply:
[10a380daeca82004dfd116401bdefa3c80a5813c]